learn/glossary.md

```diff
@@ -73,16 +129,62 @@ A static URI (or IP) that you have assigned to your node, which applications can
 
 ## Session
 
-The relationship between an Application and the Validator(s) that service it at any point in time.
+A data structure which uses data from the finality storage layer of the network to pseudo-randomly group one application with a set of nodes to provide service for a limited time. A session lasts for 4 blocks, for a total of one hour.
```

Comment:
> uses data from the finality storage layer of the network

I don't really know what this is saying. Would "data from the Pocket Network blockchain" also suffice here?

Reply:
A list of Servicers that an Application is able to send requests to. The Pocket blockchain updates the sessions so Applications receive a new, pseudo-randomly selected set of Servicers every 4 blocks (`BlocksPerSession`), approximately one hour, for each Relay Chain that the Application is staked for. During these 4 blocks, the Servicers will only be rewarded POKT for servicing Applications if they have been matched in a Session. The amount of Servicers in a Session is currently 24 (determined by the `SessionNodeCount`).